### PR TITLE
Clean up Webpack build

### PR DESCRIPTION
**Pull request by basil**
Adapts to the best practices recommended for [Development](https://webpack.js.org/guides/development/) and [Production](https://webpack.js.org/guides/production/). Untested in this repository, but similar changes were successfully tested in Pipeline: Groovy and Job DSL.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,8 +24,9 @@
     "source-map-loader": "^4.0.1",
     "ts-loader": "^9.4.2",
     "typescript": "^4.9.5",
-    "webpack": "^5.76.0",
-    "webpack-cli": "^4.10.0"
+    "webpack": "^5.76.3",
+    "webpack-cli": "^5.0.1",
```
Comment (basil) [inline package.json:28]: Upgrading to the latest versions.

File: webpack.prod.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+const { merge } = require('webpack-merge');
```
Comment (basil) [inline webpack.prod.js:1]: As recommended in [the documentation](https://webpack.js.org/guides/production/).

File: webpack.dev.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+const { merge } = require('webpack-merge');
```
Comment (basil) [inline webpack.dev.js:1]: As recommended in [the documentation](https://webpack.js.org/guides/development/).

**Review by timja — APPROVED** [review]
Thanks!

**Pull request merged**